internal/handlers/webauthn.go

```diff
@@ -28,12 +31,20 @@ func getWebAuthnUser(ctx *middlewares.AutheliaCtx, userSession session.UserSessi
 	return user, nil
 }
 
-func getWebauthn(ctx *middlewares.AutheliaCtx) (w *webauthn.WebAuthn, err error) {
-	u, err := ctx.GetOriginalURL()
-	if err != nil {
+func newWebauthn(ctx *middlewares.AutheliaCtx) (w *webauthn.WebAuthn, err error) {
+	var (
+		u       *url.URL
+		timeout time.Duration
+	)
+
+	if u, err = ctx.GetOriginalURL(); err != nil {
 		return nil, err
 	}
 
+	if timeout, err = utils.ParseDurationString(ctx.Configuration.Webauthn.Timeout); err != nil {
```

Comment:
this could be avoided if Timeout was typed as time.Duration from the beginning. It's too bad to raise this error so late. This should be handled at config validation by design.